test/compile/common.test.ts

```diff
@@ -1,72 +1,92 @@
+import {format} from '../../src/compile/axis/rules';
 /* tslint:disable:quotemark */
 
 import {assert} from 'chai';
-import {X} from '../../src/channel';
-import {numberFormat, timeFormatExpression} from '../../src/compile/common';
+import {TEXT, X} from '../../src/channel';
+import {formatSignalRef, numberFormat, timeFormatExpression} from '../../src/compile/common';
 import {defaultConfig} from '../../src/config';
 import {field, FieldDef} from '../../src/fielddef';
 import {TimeUnit} from '../../src/timeunit';
 import {NOMINAL, ORDINAL, QUANTITATIVE, TEMPORAL} from '../../src/type';
 
-describe('Common', function() {
-  describe('timeFormat()', function() {
-    it('should get the right time expression for month with shortTimeLabels=true', function() {
+describe('Common', () => {
```

Comment:
(Minor Tip) In general, you would want to separate refactoring from feature implementation so it's easier to review.